### PR TITLE
Leaderboard hit api

### PR DESCRIPTION
**Pull request by saqibjavaiddev**
In this Pull request, I continue my work on the Leaderboard list app, sending and receiving data from/to the Leaderboard API. I have completed the following project requirements.

- Read the Leaderboard API documentation to learn how to use this API.
- Create a new game with the name of your choice by using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).
- Use arrow functions instead of the function keyword.
- Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,29 +1,45 @@
...
-if (localStorage.length === 0) {
-  li += '<center>No Scores to Display</center>';
-}
-taskBox.innerHTML = li;
+
+refreshButton.addEventListener('click', loadScores);
+
+const userDataSubmit = document.getElementById('addscore');
+
+userDataSubmit.addEventListener('click', async () => {
```
Comment (danmainah) [inline src/index.js:25]: - [ ] Kindly ensure the user can add the score and after refreshing the score, the user score should appear on the list(scores data should change).

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @saqibjavaiddev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 1 above)
Comment (saqibjavaiddev) [reply src/index.js:25]: Hi @danmainah. Thanks for the review but it's already adding data and showing scores after refresh. You can check out the video below to see:

https://www.loom.com/share/4d3ef04d14524866bcf47f41dd7a8589

**Review by prolajumokeoni — APPROVED** [review]
Hi @saqibjavaiddev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**